test/modules/ModuleLib/TestResultReportingClient.cs

```diff
@@ -12,37 +15,41 @@
 
 namespace Microsoft.Azure.Devices.Edge.ModuleUtil
 {
+    using System;
+    using Microsoft.Azure.Devices.Edge.Util.TransientFaultHandling;
     using System = global::System;
-    
+
     [System.CodeDom.Compiler.GeneratedCode("NSwag", "13.1.6.0 (NJsonSchema v10.0.28.0 (Newtonsoft.Json v11.0.0.0))")]
-    public partial class TestResultReportingClient 
+    public partial class TestResultReportingClient
     {
         private string _baseUrl = "https://";
         private System.Lazy<Newtonsoft.Json.JsonSerializerSettings> _settings;
-    
+        private static RetryStrategy RetryStrategy = new ExponentialBackoff(3, TimeSpan.FromMilliseconds(100), TimeSpan.FromSeconds(3), TimeSpan.Zero, false);
```

Comment:
What's the current post-test wait time? We may need to increase the wait time there to account for the retires of the last few messages that may consider a fail otherwise.

Reply:
The max time we are retrying for is 3 seconds. I'm pretty certain wait post-test wait time is longer.